releasenotes/notes/0.10/0-10-release-8c37dadcc1c82fcc.yaml

```diff
@@ -0,0 +1,128 @@
+---
+prelude: >
+    The Qiskit Aer 0.10 release includes several performance and noise model
+    improvements. Some highlights are improved performance for parallel shot
+    GPU and HPC simulations; support for simulation of circuits
+    containing QASM 3.0 control-flow instructions; support for relaxation noise
+    on scheduled circuits in backend noise models; support of user created
+    transpiler passes for defining custom gate errors and noise models and inserting
+    them into circuits.
```

Comment:
I don't know how much you care, but there's enough elements here it might be easier to read if they're in a bullet-point list.